docs/installing-viper.rst

```diff
@@ -99,3 +115,12 @@ install and test commands
     ::
         python setup.py install
         python setup.py test
+******
```

Comment:
What do you think of using make instead?  Changing `python setup.py install` to `make` and `python setup.py test` to `make test`.